pyunifi/controller.py

```diff
@@ -222,6 +222,24 @@ def get_user_groups(self):
 
         return self._read(self.api_url + 'list/usergroup')
 
+    def get_sysinfo(self):
+        """Return basic system informations."""
+
+        return self._read(self.api_url + 'stat/sysinfo')
+
+    def get_sites(self):
+        """Return a list of all sites,
+        with their UID and description"""
+
+        if(self.version == 'v2'):
```

Comment:
The version 2 & 3 code is not needed as those versions are not supported by pyunifi any longer.  Please remove code referencing versions less than 4.

Reply:
Ok i'll do the changes next week.